showcase/tsconfig.json

```diff
@@ -5,7 +5,11 @@
     "outDir": "./dist",
     "declaration": false,
     "lib": ["dom", "dom.iterable", "es2017"],
-    "typeRoots": ["./node_modules/@types", "./node_modules/@nationalbankbelgium/stark-build/typings"],
+    "typeRoots": [
+      "./node_modules/@types",
+      "./node_modules/@nationalbankbelgium/stark-build/typings",
+      "./node_modules/@nationalbankbelgium/stark-ui/typings"
```

Comment:
We should document somewhere that the developer has to add this typing to be able to use stark-input-mask properly.

Reply:
Indeed, I'll add this to the README of Stark-UI because typings will be something global for stark-ui so it should always be added to the `tsconfig.json` of the app.